src/pkg/git/utils.go

```diff
@@ -0,0 +1,60 @@
+package git
+
+import (
+	"github.com/go-git/go-git/v5"
+	"os"
+	"path/filepath"
+)
+
+func GetGitRoot(repo *git.Repository) (string, error) {
+	wt, err := repo.Worktree()
+	if err != nil {
+		return "", err
```

Comment:
more errors wrap, make sure to catch those that aren't the first param being returned
i recommend searching for `return err` as well `return (.*), err`